blueprints/server-capabilitites.md

```diff
@@ -0,0 +1,162 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+# Server Capabilities
```

Comment:
I don't suppose we could find a better name for this? "Capabilities" is already used for user capabilities. I don't really have a better suggestion, though.

Reply:
Ehh. We discussed this quite a bit. Yeah, it's awkward that "capabilities" already exist (though those are really "permissions"). But there's not a good synonym that means exactly the same thing. There are similar things, but they all imply or omit what this actually is. It really is a "capability" of the server, something it's capable of doing.

We were hoping by calling the entire feature "Server Capabilitites," e.g. the assignments are "server_server_capabilities," it would be clearer that they're not the same. It's hard to imagine anyone being confused anyway, users and servers are totally different things.